### PR TITLE
File backport PRs from a feature branch (#3442)

### PR DESCRIPTION
**Pull request by jessebrennan**
https://github.com/DataBiosphere/azul/issues/3442

Author

- [x] PR title references issue
- [x] PR title matches issue title (preceded by `Fix: ` for bugs)   <sub>or there is a good reason why they're different</sub>
- [x] Title of main commit references issue
- [x] PR is connected to Zenhub issue and description links to issue

Author (reindex)

- [x] Added `r` tag to commit title                         <sub>or this PR does not require reindexing</sub>
- [x] Added `reindex` label to PR                           <sub>or this PR does not require reindexing</sub>

Author (freebies & chains)

- [x] Freebies are blocked on this PR                       <sub>or there are no freebies in this PR</sub>
- [x] Freebies are referenced in commit titles              <sub>or there are no freebies in this PR</sub>
- [x] This PR is blocked by previous PR in the chain        <sub>or this PR is not chained to another PR</sub>
- [x] Added `chain` label to the blocking PR                <sub>or this PR is not chained to another PR</sub>

Author (upgrading)

- [x] Documented upgrading of deployments in UPGRADING.rst  <sub>or this PR does not require upgrading</sub>
- [x] Added `u` tag to commit title                         <sub>or this PR does not require upgrading</sub>
- [x] Added `upgrade` label to PR                           <sub>or this PR does not require upgrading</sub>
- [x] Added announcement to PR description                  <sub>or this PR does not require announcement</sub>

Author (requirements, before every review)

- [x] Ran `make requirements_update`                        <sub>or this PR leaves requirements*.txt, common.mk and Makefile untouched</sub>
- [x] Added `R` tag to commit title                         <sub>or this PR leaves requirements*.txt untouched</sub>
- [x] Added `reqs` label to PR                              <sub>or this PR leaves requirements*.txt untouched</sub>

Author (before every review)

- [x] `make integration_test` passes in personal deployment <sub>or this PR does not touch functionality that could break the IT</sub>
- [x] Rebased branch on `develop`, squashed old fixups

Primary reviewer (after approval)

- [ ] Commented in issue about demo expectations            <sub>or labelled issue as `no demo`</sub>
- [ ] Decided if PR can be labeled `no sandbox`
- [ ] PR title is appropriate as title of merge commit
- [ ] Moved ticket to Approved column
- [ ] Assigned PR to an operator

Operator (before pushing merge the commit)

- [x] Checked `reindex` label and `r` commit title tag
- [ ] ~~Checked that demo expectations are clear              <sub>or issue is labeled as `no demo`</sub>~~
- [x] Rebased and squashed branch
- [x] Sanity-checked history
- [x] Pushed PR branch to Github
- [x] Branch pushed to Gitlab and added `sandbox` label     <sub>or PR is labeled `no sandbox`</sub>
- [x] Build passed in sandbox                               <sub>or PR is labeled `no sandbox`</sub>
- [x] Started reindex in `sandbox`                          <sub>or this PR does not require reindexing `sandbox`</sub>
- [x] Checked for failures in `sandbox`                     <sub>or this PR does not require reindexing `sandbox`</sub>
- [x] Added PR reference to merge commit title
- [x] Collected commit title tags in merge commit title
- [x] Moved linked issue to Merged column
- [ ] Pushed merge commit to Github

Operator (after pushing the merge commit)

- [ ] Made announcement requested by author                 <sub>or PR description does not contain an announcement</sub>
- [ ] Moved freebies to Merged column                       <sub>or there are no freebies in this PR</sub> 
- [ ] Shortened the PR chain                                <sub>or this PR is not the base of another PR</sub>
- [ ] Verified that `N reviews` labelling is accurate
- [ ] Pushed merge commit to Gitlab                         <sub>or this changes can be pushed later, together with another PR</sub>
- [ ] Deleted PR branch from Github and Gitlab

Operator (reindex) 

- [ ] Started reindex in `dev`                              <sub>or this PR does not require reindexing or does not target `dev`</sub>
- [ ] Checked for failures in `dev`                         <sub>or this PR does not require reindexing or does not target `dev`</sub>
- [ ] Started reindex in `prod`                             <sub>or this PR does not require reindexing or does not target `prod`</sub>
- [ ] Checked for failures in `prod`                        <sub>or this PR does not require reindexing or does not target `prod`</sub>

Operator

- [ ] Unassigned PR

### CODE REVIEW

File: OPERATOR.rst [hunk 1]
```diff
@@ -231,14 +231,18 @@ To do a promotion:
 Backporting from ``prod`` to ``develop``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Open a PR from the GitHub UI titled::
+#. Make a branch from ``prod`` at the last commit being backported. Title the
+   branch following this pattern::
+
+       issues/<your name>/backport-<hash of hotfix>-<another hotfix>
```
Comment (achave11) [inline OPERATOR.rst:237]: s/<your name>/<your GH handle> maybe?

**Review by achave11 — APPROVED** [review]
Looks good, only a comment with an [optional suggestion](https://github.com/DataBiosphere/azul/pull/3455/files#r714392070).

File: OPERATOR.rst (hunk 1 above)
Comment (jessebrennan) [reply OPERATOR.rst:237]: I use my name, so I don't think that's a hard and fast rule. I'll leave it as is.

File: OPERATOR.rst [hunk 2]
```diff
@@ -231,14 +231,18 @@ To do a promotion:
 Backporting from ``prod`` to ``develop``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Open a PR from the GitHub UI titled::
+#. Make a branch from ``prod`` at the last commit being backported. Title the
```
Comment (hannes-ucsc) [inline OPERATOR.rst:234]:
```suggestion
#. Make a branch from ``prod`` at the last commit being backported. Name the
```

File: OPERATOR.rst [hunk 3]
```diff
@@ -231,14 +231,18 @@ To do a promotion:
...
+   branch following this pattern::
+
+       issues/<your name>/backport-<hash of hotfix>-<another hotfix>
+
+#. Open a PR from your branch which targets ``dev``. Title it::
```
Comment (hannes-ucsc) [inline OPERATOR.rst:239]:
```suggestion
#. Open a PR from your branch which targets ``develop``. Name it::
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

File: OPERATOR.rst [hunk 4]
```diff
@@ -231,14 +231,18 @@ To do a promotion:
 Backporting from ``prod`` to ``develop``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Open a PR from the GitHub UI titled::
+#. Make a branch from ``prod`` at the last commit being backported. Name the
+   branch following this pattern::
+
+       issues/<your name>/backport-<hash of hotfix>-<another hotfix>
+
+#. Open a PR from your branch which targets ``develop``. Name it::
```
Comment (hannes-ucsc) [inline OPERATOR.rst:239]:
```suggestion
#. Open a PR from your branch, targeting ``develop``. Name it::
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
OK to merge without review once the comment below has been addressed.

**Pull request merged**